juniper_official/Protocols/Mpls/check-lsp-state-oc.rule

```diff
@@ -0,0 +1,89 @@
+/*
+ * Monitors MPLS LSP state and notifies when LSP state is UP or DOWN.
+ * One inputs control detection
```

Comment:
Change to "One input controls detection".

Reply:
Changed to "one input controls detection"